src/test/java/ebl/config/TestConfig.java

```diff
@@ -0,0 +1,23 @@
+package ebl.config;
+
+import io.restassured.RestAssured;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.Properties;
+
+public class TestConfig {
+
+  private TestConfig() {}
+
+  public static void init() throws IOException {
+    File file = new File("src/test/resources/test.properties");
+    FileInputStream fis = new FileInputStream(file);
```

Comment:
I would prefer we used the following idiom:
```java

try (FileInputStream fis = new FileInputStream(file)) {
...
}
```

(just in case someone copy-paste it elsewhere or we get more tests, which will then result in more leaked file descriptions/open files)